features/bootstrap/FilesystemContext.php

```diff
@@ -67,6 +67,19 @@ public function theClassOrSpecFileContains($file, PyStringNode $contents)
     }
 
     /**
+     * @Given the trait file :file contains:
+     */
+    public function theTraitFileContains($file, PyStringNode $contents)
+    {
+        if (version_compare(PHP_VERSION, '5.4.0', '<')) {
+            throw new PendingException('Traits implemented since PHP 5.4');
```

Comment:
You can't. Behat's own suite uses tags (e.g. @php-5.4) on the scenario and a selectively filters the tags when running